### ISSUE TITLE
Add a user option to block the boost button in case of missing ALT text

### ISSUE THREAD

**Issue by D4llo** [mastodon/mastodon#33816]
### Pitch

Add a user option to block the boost button in case of missing ALT text. I did not find a duplicate.

The idea emerge from the following thread (in french) https://pouet.chapril.org/@GythaOgg@pipou.academy/113939136471914729

### Motivation

Allow user to avoid a miss-boost of non alt content. Encourage inclusive content and a11y on the network.

**Comment by Cassolotl** [comment]
> In recent years, there has been a rush on the internet to supply image descriptions and to call out those who don’t. This may be an example of community accountability at work, but it’s striking to observe that those doing the most fierce calling out or correcting are sighted people. Such efforts are largely self-defeating. I cannot count the times I’ve stopped reading a video transcript because it started with a dense word picture. Even if a description is short and well done, I often wish there were no description at all. Get to the point, already! How ironic that striving after access can actually create a barrier. When I pointed this out during one of my seminars, a participant made us all laugh by doing a parody: “Mary is wearing a green, blue, and red striped shirt; every fourth stripe also has a purple dot the size of a pea in it, and there are forty-seven stripes—”

An excerpt from "[Against Access](https://audio.mcsweeneys.net/transcripts/against_access.html)", an essay by John Lee Clark, just under 5,000 words.
